enterprise/models.py

```diff
@@ -3008,3 +3008,28 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+class CornerstoneCourseKeys(models.Model):
```

Comment:
Suggest singular `CornerstoneCourseKey` for this model -- also, not sure if it should live in `integrated_channels/cornerstone`

Reply:
Was having some problems with circular imports when it was in there with the implementation I have at the minute, which is why I just put it in models. If you think it's important I will try to work it out!

Reply:
It would be good to understand and resolve the circular imports -- we really should be defining Cornerstone-specific models in the Cornerstone integrated channel versus at the top-level of enterprise.